zboxcore/sdk/playlist_consensus.go

```diff
@@ -34,7 +35,9 @@ func (c *playlistConsensus) AddFile(body []byte) error {
 	if ok {
 		c.consensuses[file.LookupHash].Done()
 	} else {
-		cons := &Consensus{}
+		cons := &Consensus{
+			mu: &sync.RWMutex{},
```

Comment:
do not store reference to mutex, use value, than you don't need to create default value for reference, since default value for value works well

Reply:
@dabasov  it was value, not reference. don't know who changed it, introduced this panicking issue

Reply:
@dabasov  reverted it with value